### PR TITLE
Clarify protobuf/Jackson examples in README

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: tools/build.gradle.kts [hunk 1]
```diff
@@ -24,10 +27,30 @@ plugins {
...
 dependencies {
   api(project(":cel-core"))
 
   testImplementation(platform(libs.junit.bom))
   testImplementation(libs.bundles.junit.testing)
   testRuntimeOnly(libs.junit.jupiter.engine)
 }
+
+// *.proto files taken from https://github.com/googleapis/googleapis/ repo, available as a git
+// submodule
```
Comment (XN137) [inline tools/build.gradle.kts:41]: not sure i understand what the comment is trying to tell me, what proto files are we referring to here?

File: README.md [hunk 2]
```diff
@@ -80,6 +80,50 @@ public class MyClass {
...
+package org.projectnessie.cel.toolstests;
+
+message MyPojo {
+  string Property1 = 1;
+}
+```
+
+```java
+public class MyClass {
+  public Boolean evalWIthProtobuf() {
```
Comment (XN137) [inline README.md:96]: nit: typo `With`

File: README.md [hunk 3]
```diff
@@ -80,6 +80,50 @@ public class MyClass {
...
+    ScriptHost scriptHost = ScriptHost.newBuilder().build();
+
+    Script script =
+        scriptHost
+            .buildScript("inp.Property1 == checkName")
+            .withDeclarations(
+                // protobuf types need the type's full name
+                Decls.newVar("inp", Decls.newObjectType(Dummy.MyPojo.getDescriptor().getFullName())),
```
Comment (XN137) [inline README.md:104]: might be unclear where `Dummy` is coming from, doesnt it depend on the proto filename?
in our tests its clear in the example it isnt imo

File: tools/build.gradle.kts (hunk 1 above)
Comment (snazy) [reply tools/build.gradle.kts:41]: copy-paste-mess

File: README.md (hunk 3 above)
Comment (snazy) [reply README.md:104]: Updated the readme (this "Dummy" came from the .proto file name in the test)

**Review by XN137 — APPROVED** [review]

**Pull request merged**